test/font_test.py

```diff
@@ -35,6 +35,9 @@ def setUp(self):
 
     def tearDown(self):
         pygame_font.quit()
+    
+    def get_ttf_version(self):
+        pygame_font.get_ttf_version()
```

Comment:
I know you're not done with this yet, but the reason this doesn't run is that test methods must start with `test_`

Reply:
Issue resolved.

Which are the other steps to get it working with the docs ?